hack/e2e_compare.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Compare 2 files generated by e2e_collect. This ensures elements are sorted by version
+
+type -f jq > /dev/null || {
+  echo Please install jq
+  exit 1
+}
+
+# jq would output the whole graph with elements in "nodes" sorted by "version"
+# to ensure the json output is identical for the equal graphs
+diff <(jq -S '.nodes |= sort_by(.version)' ${1}) <(jq -S '.nodes |= sort_by(.version)' ${1}) || exit 0
```

Comment:
Is one of the `${1}` is supposed to be `${2}`?

Reply:
doh! Fixed